src/api/bucket_api.js

```diff
@@ -1112,7 +1120,16 @@ module.exports = {
                             }
                         }
                     }
-                }
+                },
+                s3_policy: {
+                    $ref: 'common_api#/definitions/bucket_policy'
+                },
+                system_owner: {
```

Comment:
Maybe do this required system_owner and bucket_owner.
Since all buckets should have these values.